### PR TITLE
Built with refactored hypermedia.

### PR DESCRIPTION
**Pull request by dkorolev**

### CODE REVIEW

File: ctfo_server.h [hunk 1]
```diff
@@ -218,7 +220,8 @@ class CTFOServer final {
                               config_.Config().tick_interval_ms,
                               config_.Config().midichlorians_url_path,
                               "OK\n"),
-        storage_(current::sherlock::SherlockNamespaceName("NewCTFOStorage"), config_.Config().storage_file),
+        storage_(current::sherlock::SherlockNamespaceName("NewCTFOStorage", "TopLevelTransaction"),
```
Comment (mzhurovich) [inline ctfo_server.h]: Make these two named constants?

File: schema_base.h [hunk 2]
```diff
@@ -32,6 +32,35 @@ SOFTWARE.
 #include "../Current/TypeSystem/struct.h"
 #include "../Current/TypeSystem/enum.h"
 
+// This ugliness is to be removed, right on the next round of type evolution. -- D.K.
```
Comment (mzhurovich) [inline schema_base.h:37]: 👍

**Comment by dkorolev** [comment]
@mzhurovich PTAL

**Comment by mzhurovich** [comment]
@dkorolev 💯

**Pull request merged**